pay-api/src/pay_api/services/statement.py

```diff
@@ -457,9 +457,10 @@ def get_statement_report(statement_id: str, content_type: str, **kwargs):
             results=result_items,
         )
 
-        summary = Statement._build_statement_summary_for_methods(statement_dao, statement_purchases)
-        if summary:
-            report_inputs.statement_summary = summary
+        if extension == "pdf":
```

Comment:
already fixed in main:

<img width="1074" height="1310" alt="image" src="https://github.com/user-attachments/assets/314ff94d-c6a7-4578-a7c4-5bf296fa006a" />